src/api/checkin.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import { addAction } from '../lib/database';
+
+let app = express();
+
+app.post('/', (req, res) => {
+    addAction('CHECKIN', {
+        studentId: req.body.studentId,
+        itemAddress: req.body.itemAddress
+    }).then(() => {
+        res.successJson();
+    }).catch(e => {
+        res.failureJson(e.message);
+    });
+});
+
+export default app;
```

Comment:
This new API route should be added to the [API](https://github.com/TheFourFifths/consus/blob/dev/src/lib/api.js), with `import checkin from '../api/checkin` and `app.use('/checkin', checkin);`.